### PR TITLE
[FIX] contract_sale: Good test import

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by rousseldenis** [comment]
@MiquelRForgeFlow

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by rousseldenis** [comment]
WIP as contract_sale tests are not laucnhed

File: contract_sale/tests/test_sale_contract.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
 
 
-from odoo.odoo.tests import TransactionCase
+from odoo.tests import TransactionCase
```
Comment (MiquelRForgeFlow) [inline contract_sale/tests/test_sale_contract.py]:
```suggestion
from odoo.tests.common import TransactionCase
```

**Pull request merged**